neoutils/connection.go

```diff
@@ -50,29 +51,33 @@ func DefaultConnectionConfig() *ConnectionConfig {
 	}
 }
 
-func Connect(neoURL string, conf *ConnectionConfig) (NeoConnection, error) {
+func Connect(neoURL string, conf *ConnectionConfig, log *logger.UPPLogger) (NeoConnection, error) {
```

Comment:
It would be nice to put a comment on this method describing the method itself, the params and stating that the log is optional param.